---
layout: page
title: Bioinformatics
permalink: /Bioinformatics/
---

## Project Overview
 
<img src="/images/Bioinformatics1.png" height="180" align="center"/>

This was a collaborative project with Dr. Ryan's group at Simon Fraser University where I deployed Molecular Dynamics Simulations using GROMACS and SHAPE-Sequencing to explore the structure-functional relation in 
recurrent mutations at the 3′ UTR of NFKBIZ as a novel mechanism of oncogene deregulation and NF-κB pathway activation in the activated B-cell (ABC) subgroup. Here is the description:

* [Genome-wide discovery of somatic regulatory variants in diffuse large B-cell lymphoma](https://www.nature.com/articles/s41467-018-06354-3)
  Diffuse large B-cell lymphoma (DLBCL) is an aggressive cancer originating from mature B-cells. Prognosis is strongly associated with molecular subgroup, although the driver mutations that
  distinguish the two main subgroups remain poorly defined. Through an integrative analysis of whole genomes, exomes, and transcriptomes, we have uncovered genes and non-coding loci that
  are commonly mutated in DLBCL. Our analysis has identified novel cis-regulatory sites, and implicates recurrent mutations in the 3′ UTR of NFKBIZ as a novel mechanism of oncogene deregulation
  and NF-κB pathway activation in the activated B-cell (ABC) subgroup. Small amplifications associated with over-expression of FCGR2B (the Fcγ receptor protein IIB), primarily in the germinal
  centre B-cell (GCB) subgroup, correlate with poor patient outcomes suggestive of a novel oncogene. These results expand the list of subgroup driver mutations that may facilitate implementation
  of improved diagnostic assays and could offer new avenues for the development of targeted therapeutics.

